datadog_lambda/tracing.py

```diff
@@ -8,15 +8,17 @@
 
 from aws_xray_sdk.core import xray_recorder
 from aws_xray_sdk.core.lambda_launcher import LambdaContext
-
+ 
```

Comment:
You probably do not need this extra whitespace?

Reply:
removed and formatted correctly!